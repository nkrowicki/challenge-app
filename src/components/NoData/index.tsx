import React from 'react';

function NoResults() {
  return <p className="text dark text-center">No results found</p>;
}

export default NoResults;
