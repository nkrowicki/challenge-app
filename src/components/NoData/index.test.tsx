import React from 'react';

import { render } from '@testing-library/react';

import NoData from './index';

test('Check that NoData component shows correctly', () => {
  const { container } = render(<NoData />);
  expect(container.firstChild).toMatchSnapshot();
});
