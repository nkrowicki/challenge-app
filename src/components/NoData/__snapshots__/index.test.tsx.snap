// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check that NoData component shows correctly 1`] = `
<p
  class="text dark text-center"
>
  No results found
</p>
`;
